docs/core/reference/corectl.md

```diff
@@ -141,3 +143,64 @@ other configuration will be deleted.
 ```
 corectl reset
 ```
+
+### `grant`
+
+Command grant grants access to a policy
+for the described credentials.
+
+```
+corectl grant [-cn name] [-ou name] [-t token] [policy]
+```
+
+Flags:
+
+ * **-cn \<name>**: Common Name;
+this grants access to any valid X.509 certificate
+with the given Common Name.
+
+ * **-ou \<name>**: Organizational Unit;
+this grants access to any valid X.509 certificate
+with the given Organizational Unit.
+
+ * **-t \<token>**: Token;
+this grants access to the specified access token.
+
+If multiple flags are given,
+they will each take effect individually.
+That is, three separate grants will occur,
+not one grant with three restrictions on it.
+
+Arguments:
+
+ * **policy**: the policy to grant access to.
+See [XXX](#) for a list of policies and their meaning.
```

Comment:
The authorization guide has these listed. `../learn-more/authentication` I think.